troposphere/static/js/components/images/MyImageRequestsPage.jsx

```diff
@@ -128,8 +128,8 @@ export default React.createClass({
                 endDateText = moment(request.get("end_date")).format("MMM D, YYYY h:mm:ss a");
             }
 
-            var newMachineId = !!request.get("new_machine") ? request.get("new_machine").uuid : "N/A";
-            var newMachineProvider = !!request.get("new_machine_provider") ? request.get("new_machine_provider").name : "Unknown";
+            var newMachineId = request.get("new_machine") ? request.get("new_machine").uuid : "N/A";
+            var newMachineProvider = request.get("new_machine_provider") ? request.get("new_machine_provider").name : "Unknown";
```

Comment:
To follow-up, we have `!!` usage where we _require_ or "assert" a Boolean type. This is done often in a declaration block at the start of a method.